### PR TITLE
feat :  add home page for ecommerce

### PR DESCRIPTION
**Pull request by shruthipatil11**
add home page to ecommerce app

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,12 +5,166 @@
...
 
+                <!-- Snippet for image -->
+                <figure class="image-container badge">
+                    <img class="image--responsive" src="./src/assets/images/foot wear.jpg" alt="Responsive image" />
+                    <span>Indian wear</span>
+                </figure>
+
+                <!-- Snippet for image -->
+                <figure class="image-container badge">
+                    <img class="image--responsive" src="./src/assets/images/watch.jpg" alt="Responsive image" />
```
Comment (Maianki) [inline index.html]: You can give more meaningful names to your alt attributes for e.g if your image is a watch give ```alt="watch"```.

File: src/assets/css/styles.css [hunk 2]
```diff
@@ -1 +1,100 @@
...
+  min-width: 16rem;
+}
+
+.trending .card__body {
+  padding: 1.5px;
+}
+
+.page--center h2 {
+  text-transform: uppercase;
+  color: #3e4152;
```
Comment (Maianki) [inline src/assets/css/styles.css]: you can use CSS variables for colors.

**Review by Maianki — CHANGES_REQUESTED** [review]
Great job Shruti very well commented HTML and semantically sound. I have some suggestions
- Avoid using a descendant selector too often it might cause problems because it is recursive. So the idea here is that you avoid the descendant selector where it will cause unnecessary restrictions, specifically “high in the DOM tree”.
- Compress your images as right now some of them are above 2 MB this will affect your site performance.
These are suggestions and changes I have added in the review.

File: src/assets/css/styles.css [hunk 3]
```diff
@@ -1 +1,100 @@
...
+  grid-template-areas:
+    "header header"
+    "content content";
+  grid-template-columns: auto;
+  grid-template-rows: 7rem 1fr;
+  min-height: 100vh;
+  color: var(--LIGHT-BLACK);
+}
+
+.page main {
```
Comment (ashishkatiyar2404) [inline src/assets/css/styles.css:15]: try to use classes for styling always

**Review by ashishkatiyar2404 — APPROVED** [review]
suggested changes.
Rest looks good to me💯You can merge.
Appreciation:- UI looks really great

**Review by muskanbagrecha — APPROVED** [review]
Great UI. Can increase spacing between categories going forward.

**Review by Maianki — APPROVED** [review]
Looks good you can merge.

File: index.html (hunk 1 above)
Comment (shruthipatil11) [reply index.html]: done

File: src/assets/css/styles.css (hunk 2 above)
Comment (shruthipatil11) [reply src/assets/css/styles.css]: done

File: src/assets/css/styles.css (hunk 3 above)
Comment (shruthipatil11) [reply src/assets/css/styles.css:15]: sure, going forward will try to classes wherever needed

**Pull request merged**